src/Dscp/DB/SQLite/Class.hs

```diff
@@ -0,0 +1,16 @@
+module Dscp.DB.SQLite.Class where
+
+import Universum
+
+-- TODO: stub typeclasses, use meaningfull functions
+
+-- | Read-only interface to SQLite DB.
+class Monad m => MonadSQLiteDBRead m where
+    readMemes :: m ()
+    readMemes = pass
+
+-- | Full interface to SQLite DB.
+class MonadSQLiteDBRead m => MonadSQLiteDB m where
+    addProveOfCatsAwesomeness :: () -> m ()
+    addProveOfCatsAwesomeness () = return ()
```

Comment:
Well, why not already place here methods like `query` and `update`, with signatures like in similar methods in `sqlite-simple`? It's possible that we'll only need only them anyway.